docs/core/tools/dotnet-test.md

```diff
@@ -58,27 +61,51 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
 
 ## Options
 
-- **`-a|--test-adapter-path <PATH_TO_ADAPTER>`**
+- **`-a|--test-adapter-path <ADAPTER_PATH>`**
 
   Path to a directory to be searched for additional test adapters. Only *.dll* files with suffix `.TestAdapter.dll` are inspected. If not specified, the directory of the test *.dll* is searched.
 
 - **`--blame`**
 
   Runs the tests in blame mode. This option is helpful in isolating problematic tests that cause the test host to crash. When a crash is detected, it creates a sequence file in `TestResults/<Guid>/<Guid>_Sequence.xml` that captures the order of tests that were run before the crash.
 
+- **`--blame-crash`**
+
+  Runs the tests in blame mode and enables collecting crash dump when testhost exits unexpectedly. This option is currently only supported on Windows, and requires procdump.exe and procdump64.exe to be available in PATH. Or PROCDUMP_PATH environment variable to be set, and point to a directory that contains procdump.exe and procdump64.exe. The tools can be downloaded [here](https://docs.microsoft.com/sysinternals/downloads/procdump). Implies `--blame`.
+
+- **`--blame-crash-dump-type <DUMP_TYPE>`**
+
+  The type of crash dump to be collected. Implies `--blame-crash`.
+
+- **`--blame-crash-collect-always`**
+
+  Enables collecting crash dump on expected as well as unexpected testhost exit.
+
+- **`--blame-hang`**
+
+  Run the tests in blame mode and enables collecting hang dump when test exceeds the given timeout. Implies `--blame-hang`.
+
+- **`--blame-hang-dump-type <DUMP_TYPE>`**
```

Comment:
What are valid values for DUMP_TYPE?

Reply:
Not sure, I was just copying the descriptions from `dotnet test -h`.
But from the sdk code [here](https://github.com/dotnet/sdk/blob/9b113feabec7e29d7a9ed437fe20a9702c6853dc/src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs#L122)

looks like the valid values are `full`, `mini`, or `none`.